src/UI/RenderConfig.elm

```diff
@@ -37,6 +37,19 @@ fromWindow window =
         |> RenderConfig
 
 
+mapWindow : { window | height : Int, width : Int } -> RenderConfig -> RenderConfig
```

Comment:
this is not exactly a map, right? maybe we could rename to `updateWindowSize`, `setWindowSize` or `updateWindow`.